tests/Feature/FacebookLinkTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Interns2025b\Models\User;
+use Laravel\Sanctum\Sanctum;
+use Tests\TestCase;
+use Tests\Traits\MocksFacebookUser;
+
+class FacebookLinkTest extends TestCase
+{
+    use RefreshDatabase;
+    use MocksFacebookUser;
+
+    public function testCallbackLinksFacebookAccountSuccessfully(): void
+    {
+        $this->mockSocialiteUser(["id" => "new_fb_id"]);
+
+        $user = User::factory()->create();
```

Comment:
In all tests in this file you do `$user = User::factory()->create();`.

Instead of doing it multiple times you could just set a class field like `protected User $user;` and then fill it once like:
`protected function setUp(): void
    {
$this->user = User::factory()->create();
}
`